docs/cosmology/units.rst

```diff
@@ -0,0 +1,86 @@
+.. _astropy-cosmology-units-and-equivalencies:
+
+************************************
+Cosmological Units and Equivalencies
+************************************
+
+.. currentmodule:: astropy.cosmology.units
+
+This package defines and collects cosmological units and equivalencies.
+Many of the units and equivalencies are also available in the
+:mod:`astropy.units` namespace.
+
+We suggest importing this units package as
+
+    >>> import astropy.cosmology.units as cu
+
+
+If you want to access all units through :mod:`astropy.units`, then this units
+module can be added to the astropy.units namespace using
+:func:`~astropy.units.add_enabled_units`.
+
+    >>> u.add_enabled_units(cu)  # doctest: +SKIP
```

Comment:
For `imperial`, we have `imperial.enable()` - is that an idea here? Though if we go the only one way route, this is probably better!)

Reply:
>  Though if we go the only one way route

I don't think I understand this. Do you mean that ``cosmology.units`` can be added into the main units, but not the other way around?

Reply:
I meant would it be nice to be able to do `cosmo.units.enable()`, which, like `imperial.py` does the following:
```
def enable():
    """
    Enable Imperial units so they appear in results of
    `~astropy.units.UnitBase.find_equivalent_units` and
    `~astropy.units.UnitBase.compose`.

    This may be used with the ``with`` statement to enable Imperial
    units only temporarily.
    """
    # Local import to avoid cyclical import
    from .core import add_enabled_units
    # Local import to avoid polluting namespace
    import inspect
    return add_enabled_units(inspect.getmodule(enable))
```

But on second thought, perhaps what you have is cleaner.

Reply:
OTOH, if we ever move and deprecate ``littleh`` and ``with_H0`` from ``astropy.units``, then ``cosmo.units.enable()`` would be a convenient means to add both the units and also equivalencies into ``astropy.units``.

Reply:
I misunderstood how ``u.add_enabled_units`` worked. I thought it added the units to the astropy units namespace, but it doesn't. I'm redoing the explanation for ``u.add_enabled_units(cu)``. Now I'm also on the fence about ``enable()`` 😆. It seems convenient, but I'm not sure it adds much.

Reply:
Agreed that perhaps `enable()` is not much use, at least for now, with just a single unit added.